snuba/datasets/discover.py

```diff
@@ -65,9 +66,9 @@ def select_entity(self, query: Query) -> EntityKey:
 )
 
 
-def match_query_to_table(
+def match_query_to_entity(
```

Comment:
I would probably move `match_query_to_entity` and `track_bad_query` inside the class if they're not being used anywhere except `select_entity`. But that's a stylistic nit. Up to you.

Reply:
I'm just going to just leave it as is - these are pretty large functions, `track_bad_query` will likely be removed soon and seems easier to do this way.